gnomad/utils/release.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict, List, Optional, Union
+
+import hail as hl
+
+from gnomad.resources.grch38.gnomad import (
+    GROUPS,
+    POPS,
+    SEXES,
+    SUBSETS,
+)
+from gnomad.utils.vcf import index_globals, make_label_combos
```

Comment:
Not sure make_label_combos is used anymore 

```suggestion
from gnomad.utils.vcf import index_globals
```